packages/yew/src/lib.rs

```diff
@@ -288,6 +288,7 @@ mod server_renderer;
 pub mod suspense;
 pub mod utils;
 pub mod virtual_dom;
+pub use implicit_clone::unsync as immutable;
```

Comment:
We shouldn't re-export it. We don't do it for `gloo` and we removed the re-export for web-sys: #2084. `implicit_clone` should also not be re-exported. Any types consumed in the public API can/should be re-exported as needed.

Reply:
Ok makes sense